src/harness/harness.c

```diff
@@ -181,10 +181,13 @@ void Harness_Init(int* argc, char* argv[]) {
     } else {
         root_dir = OS_Dirname(argv[0]);
     }
-    printf("Using root directory: %s\n", root_dir);
-    result = chdir(root_dir);
-    if (result != 0) {
-        LOG_PANIC("Failed to chdir. Error is %s", strerror(errno));
+    // if root_dir is empty or ".", no need to chdir
+    if (root_dir != NULL && root_dir != '\0' && root_dir != '.') {
```

Comment:
This would disallow data in a (hidden) dotdirectory, e.g. `.carmageddon1`.
Unlikely, but still possible :)

Reply:
haha yep, ok, I'm convinced :)